sphinx_js/renderers.py

```diff
@@ -23,6 +23,8 @@ class JsRenderer(object):
 
     """
     def __init__(self, directive, app, arguments=None, content=None, options=None):
+        if not hasattr(directive.state.document.settings, 'tab_width'):
+            directive.state.document.settings.tab_width = 4
```

Comment:
Still trying to get to the bottom of why this fails. Anything against a default tab width of 8? I see that's the default in the docutils code.

Reply:
No prob, I don't really care what the default is. The default didn't noticeably effect anything.

Reply:
No prob, I don't really care what the default is. The default didn't noticeably effect anything.